timesketch/api/v1/resources/explore.py

```diff
@@ -204,6 +214,15 @@ def post(self, sketch_id):
         except KeyError:
             pass
 
+        comments = {}
```

Comment:
Why are you fetching comments here? We do that in a lazy load in the client call based on the __ts_comment special label. Or am I missing some context here?

Reply:
yes, the comment is fetched in a lazy load in the client base on each event, which works fine in the uI, but not in the API client. 

This is to make sure that if you add the `comment` field into the return values, and only then, the comment is fetched, which means you can add it into the return values of the API client.

Reply:
ok, as discussed offline - this makes sense.